main.c

```diff
@@ -370,11 +372,28 @@ int main(int argc, char **argv)
          case 'j':
              opt_jlinkarm = optarg;
              break;
+         case 'h':
+             opt_help = 1;
+             break;
          default:
              return EXIT_FAILURE;
          }
     }
 
+    if (opt_help) {
+        printf("rtt2pty: Segger RTT to PTY bridge\n");
+        printf("rtt2pty [-opt <param>]\n");
+        printf("\nOptions:\n");
+        printf("\t-d <s>            Segger device ID\n");
+        printf("\t-i <swd:jtag>     Interface\n");
```

Comment:
`-i` is not supported at the moment (it's always SWD) so I'd just remove it

Reply:
Removed it